tests/openvino/utils_tests.py

```diff
@@ -137,6 +137,7 @@
     "xglm": "hf-internal-testing/tiny-random-XGLMForCausalLM",
     "xverse": "katuni4ka/tiny-random-xverse",
     "glm4": "katuni4ka/tiny-random-glm4",
+    'open-clip': 'laion/CLIP-ViT-bigG-14-laion2B-39B-b160k'
```

Comment:
wow, I suppose this model too big for testing, can we use some tiny version?

Reply:
changed to laion/CLIP-ViT-L-14-laion2B-s32B-b82K )